alerting/src/main/java/org/opensearch/alerting/action/IndexMonitorAction2.java

```diff
@@ -0,0 +1,13 @@
+package org.opensearch.alerting.action;
+
+import org.opensearch.action.ActionType;
+
+public class IndexMonitorAction2 extends ActionType<org.opensearch.commons.model2.action.IndexMonitorResponse> {
```

Comment:
Can we have more meaningful name is it is a proxy/abstraction to monitor creator action